src/components/validation/runner/telemetry.rs

```diff
@@ -23,7 +23,7 @@ const INTERNAL_LOGS_KEY: &str = "_telemetry_logs";
 const INTERNAL_METRICS_KEY: &str = "_telemetry_metrics";
 const VECTOR_SINK_KEY: &str = "_telemetry_out";
 
-const SHUTDOWN_TICKS: u8 = 3;
+const SHUTDOWN_TICKS: u8 = 5;
```

Comment:
We're up to _five_ ticks now?! 😂

Reply:
😂  whoops I forgot I had tried changing that, before the new static sleep addition proved to get past the problem. Will change back.